docs/errors/usage_errors.md

```diff
@@ -1187,6 +1186,103 @@ except PydanticUserError as exc_info:
     assert exc_info.code == 'dataclass-on-model'
 ```
 
+## Unsupported type for `validate_call` {#validate-call-type}
+
+`validate_call` has some limitations on the callables it can validate. This error is raised when you try to use it with an unsupported callable. Currently the supported callables are functions (including lambdas), methods and instances of [`partial`][functools.partial]. In the case of [`partial`][functools.partial], the function being partially applied must be one of the supported callables.
```

Comment:
```suggestion
`validate_call` has some limitations on the callables it can validate. This error is raised when you try to use it with an unsupported callable. Currently the supported callables are functions (including lambdas) and methods and instances of [`partial`][functools.partial]. In the case of [`partial`][functools.partial], the function being partially applied must be one of the supported callables.
```